pull-request/src/person.js

```diff
@@ -11,5 +11,7 @@ Person.prototype.fullname = function() {
 }
 
 Person.prototype.wishHappyBirthday = function() {
+    this.age++;
+    console.log("Wishing happy birthday");
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Remove this logging statement. ([javascript:S2228](http://localhost:9075/coding_rules#rule_key=javascript%3AS2228))

[See it in SonarQube](http://localhost:9075/project/issues?id=training%3Apull-request&pullRequest=42&issues=AWgYy5hfUHOAr7vtaoKH&open=AWgYy5hfUHOAr7vtaoKH)

[ISSUE_KEY]: AWgYy5hfUHOAr7vtaoKH